mods/sethome/init.lua

```diff
@@ -52,6 +52,9 @@ end
 
 sethome.get = function(name)
 	local player = minetest.get_player_by_name(name)
+	if not player then
+		return nil
```

Comment:
`nil`? Consider returning something more helpful like `false, "This command can only be used from in-game!"`

Reply:
I don't see the issue with returning a helpful error message.